src/guides/components/styling.md

```diff
@@ -0,0 +1,142 @@
+:::
+
+### /packages/frontend/src/views/App.vue
+
+By default, the plugin template includes the core PrimeVue `Button` and `TextInput` components which are made available globally via `app.use(PrimeView)` in the `index.ts` file:
+
+``` ts
+<script setup lang="ts">
+// Import the Button and InputText components from the PrimeVue UI library.
+import Button from "primevue/button";
+import InputText from "primevue/inputtext";
+
+// Import the SDK interface used to interact with Caido.
+import { useSDK } from "@/plugins/sdk";
+
+// Import the ref function from Vue.js to dynamically update content.
+import { ref } from "vue";
+
+// Retrieve the SDK instance to interact with the backend.
+const sdk = useSDK();
+
+// Inital placeholder text in InputText component.
+const myVar = ref("Hello World");
+
+// Call the backend to generate a random string.
+const onGenerateClick = async () => {
+  const result = await sdk.backend.generateRandomString(10);
+  // Update placeholder text to random string.
+  myVar.value = result;
+};
+</script>
+
+// InputText component that displays value of myVar.
+<template>
+  <div class="h-full flex justify-center items-center">
+    <div class="flex flex-col gap-1">
+      <Button label="Generate random string" @click="onGenerateClick" />
+      <InputText :model-value="myVar" readonly />
+    </div>
+  </div>
+</template>
+```
+
+## Adding Components
+
+If you want to add additional components, simply import them. For example, if you wanted to display [Toast](https://primevue.org/toast/) messages:
+
+### /packages/frontend/src/views/App.vue
+
+::: warning
+In the examples below, comments are just used to highlight changes. Do not actually comment out these lines.
+:::
+
+Add the necessary imports:
+
+``` ts
+// <script setup lang="ts">
+// import Button from "primevue/button";
+// import InputText from "primevue/inputtext";
+import Toast from "primevue/toast";
+import { useToast } from "primevue/usetoast";
+```
+
+Then add the hook to provide the API:
+
+``` ts
+// const sdk = useSDK();
+const toast = useToast();
+```
+
+Change the `onGenerateClick` button handler function to generate a Toast message with your desired options:
+
+``` ts
+// const onGenerateClick = async () => {
+  toast.add({
+    severity: "info",
+    summary: "Info",
+    detail: "Button clicked!",
+    life: 3000
+  });
+  // const result = await sdk.backend.generateRandomString(10);
+  // myVar.value = result;
+// };
+```
+
+Then, include the Toast component in the template:
+
+``` ts
+// <template>
+  // <div class="h-full flex justify-center items-center">
+    <Toast />
+    // <div class="flex flex-col gap-1">
+      // <Button label="Generate random string" @click="onGenerateClick" />
+      // <InputText :model-value="myVar" readonly />
+    // </div>
+  // </div>
+// </template>
+```
+
+### /packages/frontend/src/index.ts
```

Comment:
This section should be the first one under the "Adding a Toast Notification" section, since the logical progression for this guide is:
- Registering the ToastService
- Using the Toast Component